migrations/1757575120.sh

```diff
@@ -0,0 +1,5 @@
+echo "Add git defaults"
+
+omarchy-refresh-config git/config
+
+source $OMARCHY_PATH/install/config/git.sh
```

Comment:
Don't think we want to overwrite someone's existing config. So let's just drop the migration and this is a new default going forward only.